tests/test_utils.py

```diff
@@ -364,3 +365,22 @@ def test_project_files_scanning(self):
         with six.assertRaisesRegex(self, exceptions.MalformedConfigFile, msg):
             for file, lang in utils.get_project_files(os.getcwd(), expression):
                 pass
+
+class GitUtilsTestCase(unittest.TestCase):
+
+    def test_fetch_timestamp_from_git_tree(self):
+        # A bit meta, lets try to get the timestamp of the
+        # current file
+        epoch_ts = utils.get_git_file_timestamp(
+            os.path.dirname(os.path.abspath(__file__))
+        )
+        self.assertIsNotNone(epoch_ts)
+
+    def git_timestamp_is_parsable(self):
```

Comment:
Rename `git_timestamp_is_parsable()` to `test_git_timestamp_is_parsable()`  - it is not run automatically by tests.